README.md

```diff
@@ -10,6 +10,25 @@ All docs are written in markdown format. The most up-to-date docs are in the `cu
 
 Feel free to add an issue if there is documentation that you'd like to see. See the [contributing guide](https://github.com/substrate-developer-hub/knowledge-base/blob/master/CONTRIBUTING.md) for guidance on making PRs.
 
+## Directory Structure
+
+The knowledge base has six main categories. These are not meant to be a table of contents or
+imposed order in which to read documentation, but rather groups of articles within the same context
+and for similar audiences.
+
+1. **Getting Started:** Entry-level documentation. For example dependencies, installation, key
+  terms.
+1. **Learn Substrate:** Conceptual documentation. Articles about assumptions, key concepts, and
+  design decisions in Substrate that will help you make educated decisions about developing your
+  chain.
+1. **Runtime:** Documentation that goes more in-depth and covers topics specific to runtime
+  development like macros, storage, fees, and pallet architecture.
+1. **Smart Contracts:** A section dedicated to smart contracts on Substrate.
+1. **Integrate:** Documentation about integrating and interacting with your chain for node
+  operation, key management, and front-end development.
+1. **Advanced:** Sometimes you need more details than just the abstraction. The Advanced section
+  contains information about encoding/decoding, consensus, cryptography, etc.
```

Comment:
This seems like it could turn into the `misc` category. Icould also see encoding/decoding fitting into `integrate` or into `runtime` because storage API uses scale codec. Consensus could fit into one called client.

Reply:
Yeah, agree, the lines get a bit blurry. "Misc" doesn't sound like a very interesting category though :)